src/docs/src/api/database/find.rst

```diff
@@ -1401,7 +1401,9 @@ it easier to take advantage of future improvements to query planning
     :>json object mrargs: Arguments passed to the underlying view.
     :>json number limit: Limit parameter used.
     :>json number skip: Skip parameter used.
-    :>json array fields: Fields to be returned by the query.
+    :>json array fields: Fields to be returned by the query.  The `[]`
+       value here means all the fields since there is no projection
+       happens in that case.
```

Comment:
Minor grammar nit: wonder if `there is no projection happens` might be better as `there is no projection happening`?

Another nit: maybe a comma after `fields` and before `since`

Reply:
Aw, right.  The first one is actually grammar mistake, thanks for spotting that.